CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ project(cdt)
 set(VERSION_MAJOR 3)
 set(VERSION_MINOR 1)
 set(VERSION_PATCH 0)
-set(VERSION_SUFFIX rc1)
+set(VERSION_SUFFIX "")
```

Comment:
We comment this out in leap. I don't know if setting it to `""` is valid.

Reply:
I checked and it works:
```
cdt-cpp --version
cdt-cpp version 3.1.0
```